HandledError.py

```diff
@@ -0,0 +1,15 @@
+from flask import jsonify
+
+
+class HandledError(Exception):
```

Comment:
Please follow Python files naming convention. It should be like some_module.py, not SomeModule.py, though classes are named like MyClass

Reply:
fixed